lib/Cronjob/Sync.php

```diff
@@ -5,121 +5,202 @@
 use FriendsOfRedaxo\Neues\Author;
 use FriendsOfRedaxo\Neues\Category;
 use FriendsOfRedaxo\Neues\Entry;
+use rex;
 use rex_cronjob;
 use rex_i18n;
+use rex_media;
+use rex_media_service;
+use rex_path;
 use rex_socket;
+use rex_sql;
+use rex_string;
 
 class Sync extends rex_cronjob
 {
-    /** @var array<string,string> */
-    private $rest_urls = ['category' => '/rest/neues/category/5.0.0/',
-        'author' => '/rest/neues/author/5.0.0/',
-        'entry' => '/rest/neues/entry/5.0.0/'];
-
-    /**
-     * @return bool
-     */
+    private static $ENDPOINT = '/rest/neues/entry/5.0.0/';
```

Comment:
Soll das eine Variable sein (dann $camelCase) oder eine Konstante (dann KONSTANTE). Ich würde es als 
```php
private const ENDPOINT = ...
```
schreiben

Reply:
Du kannst beim Review direkt Suggestions einfügen.

```suggestion
    private const ENDPOINT = '/rest/neues/entry/5.0.0/';
```

![image](https://github.com/user-attachments/assets/e697c664-6b1d-49c3-86ed-eb313bccdeaa)

In diesem Fall ist klar, dass ich natürlich auch an anderer Stelle ersetzen muss.